packages/blockfrost/test/e2e/queryTransactions.test.ts

```diff
@@ -26,5 +26,115 @@ describe('blockfrostWalletProvider', () => {
         txId: '2db6592c4782064295295b365c2e8ce84084fa24b1b3f5834f3c6b65268b7878'
       });
     });
+    // TODO: waiting for fix in db-sync/blockfrost
+    // it('has collaterals for failed contract', async () => {
```

Comment:
You can `it.skip` instead of commenting out if you wish. Then we have visibility of these in the report.